validator/src/main/java/com/amazon/aoc/validators/ContainerInsightStructuredLogValidator.java

```diff
@@ -46,8 +47,12 @@ void init(Context context, FileConfig expectedDataTemplate) throws Exception {
       FileConfig fileConfig = new LocalPathExpectedTemplate(FilenameUtils.concat(
           expectedDataTemplate.getPath().toString(),
           logType + ".json"));
-      String templateInput = mustacheHelper.render(fileConfig, context);
-      schemasToValidate.put(logType, parseJsonSchema(templateInput));
+      try {
+        String templateInput = mustacheHelper.render(fileConfig, context);
```

Comment:
Doesn't the IOException happen a couple lines earlier where the FileConfig is created?

Reply:
It occurs here when we are trying to parse the json

Reply:
Is there a way we can make so it only parses the files we want it to parse?

Reply:
I think another way to do this would be to just read all the JSON files present in the directory.

Reply:
A previous implementation I had was including the expected JSON files for each test case in an array and then checking which test case is currently being ran to select the correct JSON (only 1 other integration test utilizes this log validator). However, I do think that a try catch is a good solution since it allows the test to still run; however the test will fail if it does not detect the desired logs either way

Reply:
@jefchien what do you think of letting this in for now and creating an issue to change later. @alolita wanted us to add tests for the added features in this release. So this will allow us to get testing in quickly.

Reply:
I don't have a problem with the current implementation.